src/domain/occurrence/query.ts

```diff
@@ -17,6 +17,26 @@ export const QUERY_OCCURRENCE = gql`
     startTime
     endTime
     placeId
+    linkedEvent {
+      offers {
+        isFree
+        description {
+          fi
+          sv
+          en
+        }
+        price {
+          fi
+          sv
+          en
+        }
+        infoUrl {
+          fi
+          sv
+          en
+        }
+      }
```

Comment:
can replaced with offerFields fragment?

```
offers {
        ...offerFields
      }
```

Reply:
I used offerFields, but also improved other parts by using field fragments of image and translations.